src/types.ts

```diff
@@ -56,14 +56,21 @@ export interface IOnChainPayment {
 }
 
 export interface ITransaction {
+  created_at: number // unix 
   amount: number
+  sat: number,
+  usd: number,
   description: string
-  created_at: Date
-  hash?: string
-  destination?: string
   type: TransactionType
+  hash?: string
+  fee: number,
+  feeUsd: number,
+  username: string,
+  // destination?: string
```

Comment:
Why is this being removed?

Reply:
not being used currently. destination is meant for push payment to other wallet but this is just an idea I have been working  on months back that would need a lot more thoughts.